torsiondrive/qm_engine.py

```diff
@@ -144,6 +145,44 @@ def optimize_geomeTRIC(self):
     def load_native_output(self):
         return Molecule()
 
+class EngineOpenMM(QMEngine):
+    def load_input(self, input_file):
+        """Input file is the name of the pdb file with the coords in we also require that the xml has the same name"""
+
+        self.m_pdb = Molecule(input_file)[0]
+        self.M = copy.deepcopy(self.m_pdb)
+
+        xml_name = os.path.splitext(input_file)[0] + '.xml'
+        # Check the xml file is present
+        assert os.path.exists(xml_name) is True, "OpenMM requires a pdb and xml file, ensure you have both in the current folder with the same prefix"
+        with open(xml_name) as f:
+            self.xml = f.read()
+
+    def write_input(self):
+        """write the pdb and xml file for geometric assuming the rename is UNK"""
```

Comment:
This doc string is not accurate any more. I think its better to say:
```
write a new pdb file with updated geometry and the original xml file
```

Reply:
changed.